server/graphql/v2/interface/Account.js

```diff
@@ -351,6 +362,14 @@ const accountFieldsDefinition = () => ({
         offset: args.offset,
       };
 
+      if (args.dateFrom) {
+        query.where['createdAt'] = { [Op.gte]: args.dateFrom };
+      }
+      if (args.dateTo) {
+        query.where['createdAt'] = query.where['createdAt'] || {};
+        query.where['createdAt'][Op.lte] = args.dateTo;
```

Comment:
No need to update, this could also be:
```suggestion
        query.where['createdAt'] = Object.assign({}, query.where['createdAt'], { [Op.lte]: args.dateTo });
```

Reply:
@kewitz : Ah, thanks. Seems more nicer this way. 😄